lib/recurly/subscription/add_ons.rb

```diff
@@ -27,7 +27,7 @@ def << add_on
 
         exist = @add_ons.find { |a| a.add_on_code == add_on.add_on_code }
         if exist
-          exist.quantity ||= 1 and exist.quantity += 1
+          exist.quantity ||= 1 and exist.quantity += add_on.quantity || 1
```

Comment:
Do you understand what the original intention of the `and` was here? As far as I can tell this should just be on two lines.